src/global-firestore-data.ts

```diff
@@ -215,6 +215,7 @@ export const setOnboardingData = (name: FirestoreUserName, onboarding: AppOnboar
       colleges: [{ acronym: onboarding.college }],
       majors: onboarding.major.map(acronym => ({ acronym })),
       minors: onboarding.minor.map(acronym => ({ acronym })),
+      grad: onboarding.grad.map(acronym => ({ acronym })),
```

Comment:
Should this look like the the college line, since we're only going to add one grad program? So `[{ acronym: onboarding.grad}]`

Reply:
Oh, I also think I called this "program" instead of grad in my PR, but when combining our code I can make sure we are consistent! (I prefer grad anyway)

Reply:
So the onboarding.college type is a string, which is why it says [{ acronym: onboarding.college }]. If I made grad a string, then we would always have to assign it a value. If it's a string[], then it can be empty (the same way the minors array would be empty if they didn't have a minor).